examples/lennard_jones_potential.py

```diff
@@ -34,7 +34,7 @@
         hopping_calculator=InteractingHoppingCalculator(
             baserate=SquareBaseRate(straight_rate=0.01, diagonal_rate=0.002),
             temperature=200,
-            lattice_spacing=2.5,
+            lattice_properties=(2.5, np.array([1, 0]), np.array([0, 1])),
```

Comment:
params should maybe match the layout of LineDefect - so have lattice_directions too?